### ISSUE TITLE
Refactoring proposal

### ISSUE THREAD

**Issue by aviggiano** [SunWeb3Sec/DeFiHackLabs#607]
Hello,

I am starting to learn this codebase and I believe it could benefit from a structural refactor.

Currently, the README contains descriptions from various exploits, in chronological order, together with foundry reproducers. The issue is that the reproducers are all on the `src` folder, without any ordering whatsoever.

I propose a slight change on that structure:

```
images
  |_ ...
academy
  |_ ...
hacks
  |_ 2024
    |_ 01
      |_ OrbitChain_exp.sol
```

If you agree on this refactor, I can submit a PR

**Comment by SunWeb3Sec** [comment]
Hey, it makes sense to me. However, we can wait one day in case anyone has objections.
Thanks a lot!

**Comment by SunWeb3Sec** [comment]
@aviggiano I have closed #547 , you can go ahead. thanks a lot.
